src/components/DropZone/DropZone.scss

```diff
@@ -132,13 +140,11 @@ $dropzone-stacking-order: (
   justify-content: center;
   align-items: center;
   padding: $dropzone-padding;
-  border: border-width(thick) $dropzone-border-style var(--p-border);
-  border-radius: calc(
-    var(--p-border-radius-base) + #{border-width(thick)} + #{rem(1px)}
-  );
+  border: border-width(thick) $dropzone-border-style var(--p-interactive);
   text-align: center;
-  color: var(--p-text);
-  background-color: var(--p-surface-hovered);
+  color: var(--p-interactive);
+  background-color: var(--p-surface-selected);
+  pointer-events: none;
```

Comment:
Fixes a bug where the inner nested drop zone was not available during a dragging event.